updater/update.py

```diff
@@ -2,15 +2,26 @@
 from run_models import run_models
 import os
 import shutil
+import sys
 
 if __name__ == "__main__":
 
-    print("Downloading Data")
-    download_data("../shared_config/data_sources.json", "../data/downloads")
+    # // This is helpful for testing
+    devmode = True if sys.argv[1] == "devmode" else False
+    data_sources = (
```

Comment:
Is this needed permanently?

Reply:
If we aren't adding anymore datasets yet, we can remove it.